docs/articles/nunit/writing-tests/attributes/fixturelifecycle.md

```diff
@@ -0,0 +1,38 @@
+---
+uid: fixturelifecycleattribute
+---
+
+# FixtureLifeCycle
+
+Added in **NUnit 3.13.**

```

Comment:
I'd like to start adding this for any new feature that is added after the 3.0 release so users know which version the documentation applies to. If anyone has a better idea of how to format this, please post suggestions.

Reply:
It looks very nice!

Reply:
This LGTM 👍 . Ideally we'd have different docs for various releases, but at the moment traceability like this is our best option, and will help us in the future.

Reply:
I wouldn't mind a dropdown at the top of each page that says what release it is for, but a separate set of docs became a bit of a problem with the NUnit 2.x docs. People linked to the current release at the time, so Google searches ended up pointing to old documentation and people ended up missing updates in newer docs.